dids/didjwk.go

```diff
@@ -0,0 +1,117 @@
+package dids
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+
+	"github.com/tbd54566975/web5-go/crypto"
+	"github.com/tbd54566975/web5-go/crypto/dsa"
+	"github.com/tbd54566975/web5-go/jwk"
+)
+
+type newDIDJWKOptions struct {
+	keyManager  crypto.KeyManager
+	algorithmID string
+}
+
+type newDIDJWKOption func(o *newDIDJWKOptions)
+
+// KeyManager is an option that can be passed to NewDIDJWK to provide a KeyManager
+func KeyManager(k crypto.KeyManager) newDIDJWKOption {
```

Comment:
It's generally bad form to return a private type from a public function. The option type itself can be public without concern for violating encapsulation, because the function parameter is private.

Reply:
done ya ninny: https://github.com/TBD54566975/web5-go/commit/292191160446308352e415efe2a16772e6bf1f28